cdap-ui/cypress/integration/controlcenter.spec.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as Helpers from '../helpers';
+
+let headers = {};
+const testPipeline = `test_pipeline_${Date.now()}`;
+
+describe('Using control center to see and filter things', () => {
+  before(() => {
+    Helpers.loginIfRequired().then(() => {
+      cy.getCookie('CDAP_Auth_Token').then((cookie) => {
+        if (!cookie) {
+          return;
+        }
+        headers = {
+          Authorization: 'Bearer ' + cookie.value,
+        };
+      });
+    });
+  });
+  before((done) => {
+    // Deploy a pipeline to have a pipeline and some datasets to see
+    Helpers.deployAndTestPipeline('fll_airport_pipeline2.json', testPipeline, done);
+  });
+  after(() => {
+    // Delete the pipeline to clean up
+    cy.cleanup_pipelines(headers, testPipeline);
+  });
+  it('allows users to open filter menu and filter out pipelines', () => {
+    cy.visit('/cdap/ns/default/control');
+    cy.wait(2000);
```

Comment:
we should stop doing explicit waits. `cy.get` have an implicit wait until the element shows up. There is a timeout that you can override on the get method.

Reply:
Agreed. That said, I tried adding an implicit wait to the cy.get here, and for some reason this is not working. I was able to remove the second explicit wait though, as well as a wait in the deployment of the pipeline. 

I added a to-do to comment to further investigate this and other oddities in this test later.